health-services/project-factory/src/server/utils/onGoingCampaignUpdateUtils.ts

```
@@ -0,0 +1,806 @@
+    });
+  }
+}
+
+
+async function finalizeAndUpload(newWorkbook: any, mappingObject: any, resource: any) {
+  const responseData = await createAndUploadFile(newWorkbook, mappingObject, mappingObject?.CampaignDetails?.tenantId);
+  const fileStoreId = responseData?.[0]?.fileStoreId;
+  const resourceDetails = (await getResourceFromResourceId(mappingObject, [resource.createResourceId], resource))[0];
+  resourceDetails.processedFilestoreId = fileStoreId;
+  resourceDetails.processedFileStoreId = resourceDetails.processedFilestoreId;
+  delete resourceDetails.processedFilestoreId;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)